.gitignore

```diff
@@ -1,2 +1,3 @@
 Cargo.lock
 target/
+logs/
```

Comment:
I think that lists like these in general should be kept sorted, so logs/ before target/